packages/mobile/src/screens/user-list-screen/FollowersScreen.tsx

```diff
@@ -27,19 +18,22 @@ export const FollowersScreen = () => {
   const { params } = useProfileRoute<'Followers'>()
   const { userId } = params
   const { data: currentUserId } = useCurrentUserId()
-  const dispatch = useDispatch()
 
-  const handleSetFollowers = useCallback(() => {
-    dispatch(setFollowers(userId))
-  }, [dispatch, userId])
+  const { data, hasMore, isLoadingMore, loadMore, isLoading } = useFollowers({
```

Comment:
dang we could even just spread this directly into the props

Reply:
realizing do we want to do this actually? im afraid it may lead to more rerenders? its very tempting to merge

Reply:
im going to merge for now but note to both of us to watch out for this